source/tutorials.rst

```diff
@@ -0,0 +1,122 @@
+Tutorials
+=========
+
+
+Getting started with ``pytest_container``
+-----------------------------------------
+
+.. note::
+
+   In this tutorial we will start with a minimal python project and start using
+   ``pytest_container`` to test its command line interface.
```

Comment:
_The following tutorial demonstrates how to_ would be better methinks.

- What does _its_ refer to?
- Command-line interface